schema/event/createEventStep2.schema.ts

```diff
@@ -0,0 +1,35 @@
+import { date, number, object, string } from "yup"
+
+export const CreateEventStep2Schema = object({
+  startDate: date()
+    .typeError("Invalid date")
+    .required("Event start date cannot be empty"),
+  endDate: date()
+    .typeError("Invalid date")
+    .required("Event end date cannot be empty"),
+  doorTime: date()
+    .typeError("Invalid date")
+    .required("Event door time cannot be empty"),
+  location: object()
```

Comment:
Does this allow url such as Zoom link?

Reply:
right now, no. It's a premade-component I use. Maybe gotta add a switch to toggle between physical and online event for them to input a link.

Reply:
will do in the next PR